gui/wxpython/lmgr/statusbar.py

```diff
@@ -0,0 +1,144 @@
+"""
+@package frame.statusbar
+
+@brief Classes for main window statusbar management
+
+Classes:
+ - statusbar::SbMain
+ - statusbar::SbMask
+
+(C) 2022 by the GRASS Development Team
+
+This program is free software under the GNU General Public License
+(>=v2). Read the file COPYING that comes with GRASS for details.
+
+@author Linda Kladivova <lindakladivova gmail.com>
+@author Anna Petrasova <kratochanna gmail.com>
+@author Vaclav Petras <wenzeslaus gmail.com>
+"""
+
+import wx
+
+from core.gcmd import RunCommand
+from gui_core.wrap import Button
+from grass.script import core as grass
+
```

Comment:
Please do the import differently (this was an old way), and adjust grass -> gs throughout the file
```suggestion

import grass.script as gs

from core.gcmd import RunCommand
from gui_core.wrap import Button

```